test/network/2020-03-01/armnetwork/zz_generated_virtualwans.go

```diff
@@ -30,32 +30,32 @@ func NewVirtualWansClient(con *armcore.Connection, subscriptionID string) *Virtu
 }
 
 // BeginCreateOrUpdate - Creates a VirtualWAN resource if it doesn't exist else updates the existing VirtualWAN.
-func (client *VirtualWansClient) BeginCreateOrUpdate(ctx context.Context, resourceGroupName string, virtualWanName string, wanParameters VirtualWan, options *VirtualWansBeginCreateOrUpdateOptions) (VirtualWanPollerResponse, error) {
-	resp, err := client.createOrUpdate(ctx, resourceGroupName, virtualWanName, wanParameters, options)
+func (client *VirtualWansClient) BeginCreateOrUpdate(ctx context.Context, resourceGroupName string, virtualwanName string, wanParameters VirtualWAN, options *VirtualWansBeginCreateOrUpdateOptions) (VirtualWANPollerResponse, error) {
```

Comment:
For the type WAN is in caps but for the param name it's all lower case

Reply:
This is (at present) by design.  Param names always start all lower-case which is the convention unless it collides with a key-word or package name (e.g. param "url" would have to be "URL").

Reply:
Oh I was referring to virtualwanName, I understand the naming for wanParameters.

Reply:
Fixed